web/src/engine/websites/AsuraScans.ts

```diff
@@ -47,8 +58,40 @@ export default class extends DecoratableMangaScraper {
     }
 
     public override async FetchChapters(manga: Manga): Promise<Chapter[]> {
```

Comment:
The chapters are directly available as HTML, any reason why extracting these from the scripts which is likely much more error prone?

![image](https://github.com/user-attachments/assets/be058fa8-abc7-4a43-8f5f-5d59a853da82)

Reply:
Its precisely because those selectors are changing very often that i would rather extract chapters from json structure and that scrapers have troubles. The selectors on React websites are very hard to chose the class names are too generic.